dnsmasq/default.toml

```diff
@@ -0,0 +1,7 @@
+[[nameservers]]
+domain = "fooxample.com"
```

Comment:
This should be "example.com". example.com is a domain set aside by the IANA for testing purposes.